requirements.txt

```diff
@@ -5,4 +5,5 @@ jsonpickle==2.1.0
 numpy==1.22.2
 pyyaml==6.0
 scipy==1.8.0
+toml
```

Comment:
shouldn't we have a version here?

Reply:
Yes. It is definitively safer to increase the version of the dependencies manually. Thanks